cover_agent/PromptBuilder.py

```diff
@@ -125,6 +127,7 @@ def build_prompt(self) -> dict:
             "source_file_name": self.source_file_name,
             "test_file_name": self.test_file_name,
             "source_file_numbered": self.source_file_numbered,
+            "test_file_numbered": self.test_fule_numbered,
```

Comment:
Spelling error: `fule` but should be `file`. Need to refactor.

Reply:
fixed